jaseci_core/jaseci/actions/live_actions.py

```diff
@@ -4,21 +4,29 @@
 from importlib.util import spec_from_file_location, module_from_spec
 from jaseci.utils.utils import logger
 from jaseci.actions.remote_actions import ACTIONS_SPEC_LOC
+from jaseci.actions.remote_actions import serv_actions, mark_as_remote
 import requests
+import inspect
 
 live_actions = {}
 
 
-def jaseci_action(act_group=None):
+def jaseci_action(act_group=None, aliases=list(), allow_remote=False):
     """Decorator for Jaseci Action interface"""
+    caller_globals = dict(inspect.getmembers(
+        inspect.stack()[1][0]))["f_globals"]
+    if allow_remote and 'serv_actions' not in caller_globals:
+        caller_globals['serv_actions'] = serv_actions
```

Comment:
I am curious what does this `caller_globals` do and why is this change conditioned if the action is allowed to be remote or not?

Reply:
Yea, you'd have to use that function serve_actions.